godel_scan_analysis/include/godel_scan_analysis/scan_utilities.h

```diff
@@ -0,0 +1,52 @@
+#ifndef SCAN_UTILITIES_H
+#define SCAN_UTILITIES_H
+
+#include <vector>
+
+namespace rms
+{
+
+template<typename FloatType>
+struct Point
+{
+  FloatType x,y;
+};
+
+template<typename FloatType>
+struct LineCoef
+{
+  FloatType slope_;
+  FloatType intercept_;
+
+  LineCoef(double slope, double intercept)
+    : slope_(slope)
+    , intercept_(intercept)
+  {}
+
+  FloatType operator()(const FloatType x) const
```

Comment:
Since this struct has some functionality maybe it should be a class? See [here](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Structs_vs._Classes) for guidelines on struct uses